skll/learner.py

```diff
@@ -2412,7 +2413,7 @@ def cross_validate(self,
                 kfold = StratifiedKFold(n_splits=cv_folds)
                 cv_groups = None
             else:
-                kfold = KFold(n_splits=cv_folds, random_state=random_state)
+                kfold = KFold(n_splits=cv_folds)
```

Comment:
If we're no longer passing in the random state, how is it picking it up?

Reply:
According to the [docs](https://scikit-learn.org/stable/modules/generated/sklearn.model_selection.KFold.html#sklearn.model_selection.KFold), `random_state` is only used if the `shuffle` keyword argument for `Fold` is `True` (it's `False` by default). With v0.22, setting `random_state` raises a warning unless `shuffle` is `True` so I removed it. Just to confirm, I double-checked the [0.21.3 documentation](https://scikit-learn.org/0.21/modules/generated/sklearn.model_selection.KFold.html#sklearn.model_selection.KFold) and `shuffle` was `False` by default then too.